BraintreeCore/src/main/java/com/braintreepayments/api/DeviceInspector.kt

```diff
@@ -16,41 +14,42 @@ import androidx.annotation.VisibleForTesting
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
 class DeviceInspector @VisibleForTesting internal constructor(
     private val appHelper: AppHelper,
-    private val uuidHelper: UUIDHelper,
     private val signatureVerifier: SignatureVerifier,
 ) {
+
     constructor() : this(
         AppHelper(),
-        UUIDHelper(),
         SignatureVerifier(),
     )
 
     internal fun getDeviceMetadata(
         context: Context?,
+        configuration: Configuration?,
         sessionId: String?,
-        integration: String?,
+        integration: String?
     ): DeviceMetadata {
         return DeviceMetadata(
```

Comment:
Do we have a ticket for adding `link_type` to Android, like we did on iOS (https://github.com/braintree/braintree_ios/pull/1198)?

Reply:
Ticket is closed at the moment (DTBTSDK-3596), happy to add it, just want to check with you if necessary. 
@jaxdesmarais do you have more context about it?

Reply:
Yes, we had not added that value initially because it would require an Arachne change to ingest the link type. Now that we are hitting FPTI directly we can make this change. Created ticket DTBTSDK-3704 for this work and put it in the sprint after the one that starts tomorrow.